packages/app-compat/src/public-types.ts

```diff
@@ -85,7 +85,7 @@ export interface FirebaseNamespace {
      *
      * DO NOT call this constuctor directly (use firebase.app() instead).
      */
-    App: FirebaseAppContructor;
+    App: FirebaseAppConstructor;
```

Comment:
As noted in the initial PR, this is the only change that might be an api change. Given the comment here, I'd hope it isn't particularly problematic. Anyone upgrading would just need to fix the spelling of the type.